src/Webform.js

```diff
@@ -1128,13 +1128,23 @@ export default class Webform extends NestedDataComponent {
       this.refs.errorRef.forEach(el => {
         this.addEventListener(el, 'click', (e) => {
           const key = e.currentTarget.dataset.componentKey;
-          this.focusOnComponent(key);
+          if (!this.focusOnComponent) {
```

Comment:
Why is this method not defined?  Is the "this" pointer not the webform class?  This seems like a strange change to get around a different problem.

Reply:
Yes, "this" indicates to the component which caused error. Not the webform.

Reply:
But this is called within an arrow function... wouldn't that inherit context from the bounding context (ie webform instance)?